wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -352,6 +352,12 @@ public void removeWorkspace(String workspaceId) throws ConflictException, Server
      * @param accountId
      *         account which should be used for this runtime workspace or null when
      *         it should be automatically detected
+     * @param isRestore
```

Comment:
I don't think that `isRestore` is a good  name, please change it to something like`restore`, `trueIfRestore`.